server.js

```diff
@@ -41,3 +45,34 @@ io.on('connection', function(client){
     ClientManagerService.disconnected(client);
   });
 });
+function getGameServerPath(){
+    if (fs.existsSync('./config/config.json')) {
+        console.log("Checking your config file");   
+        var config;
+        fs.readFile('./config/config.json', function read(err, data) {
+          if (err) {
+            throw err;
+          }
+        config = data;
```

Comment:
How about just `var config = data;`, and remove the `var config;` a few lines above?